### PR TITLE
Add an Examine to Bonfires

### PR DESCRIPTION
**Pull request by lbnesquik**
This just let you check out how much fuel is in there.

### CODE REVIEW

File: code/game/objects/structures/bonfire.dm [hunk 1]
```diff
@@ -34,6 +34,14 @@
 /obj/structure/bonfire/permanent/sifwood/New(newloc, material_name)
 	..(newloc, MAT_SIFWOOD)
 
+/obj/structure/bonfire/examine(mob/user)
+	var/X = get_fuel_amount()
+	user << "The fire has [X] burning logs in it."
+	if(grill)
+		user << "The [src] has a crude grill plate over it."
+	if(stake)
+		user << "The [src] has a makeshift stake plate over it, perfect for witches and space templars."
```
Comment (TheFurryFeline) [inline code/game/objects/structures/bonfire.dm]: Undefined var for stake here. Also, please use to_chat per the new standard.

Comment (TheFurryFeline) [reply code/game/objects/structures/bonfire.dm]: Also, this is editing a core file. If you can, ya should make a _ch file addition or add `//CHOMPStation Addition Start/End` at either end of the new code block.

Comment (ExabyteDiscord) [reply code/game/objects/structures/bonfire.dm]: Seconding what @TheFurryFeline said. Definitely going to need these changes before the merge could be accepted.

**Comment by lbnesquik** [comment]
@ExabyteDiscord REquested changes have been made

**Pull request merged**